Makefile.mjs

```diff
@@ -0,0 +1,60 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
is this needed if `execa` is declared in the `package.json` file?

Reply:
As per discussion: It complains that execa is only included in `devDependencies`, which is fine, as it **_is_** a dev dependency